sycl/source/detail/device_info.hpp

```diff
@@ -493,6 +493,23 @@ template <> struct get_device_info<device, info::device::parent_device> {
   }
 };
 
+// Specialization for bfloat16
+template <>
+struct get_device_info<bool, info::device::ext_intel_bf16_conversion> {
+  static bool get(RT::PiDevice dev, const plugin &Plugin) {
+    // TODO: We claim, that all Intel GPU devices support bfloat16 conversion
+    // feature but we'd better have a low-level extension to query for support
+    // of the feature
+    platform plt =
+        get_device_info<platform, info::device::platform>::get(dev, Plugin);
+    std::string platform_name = plt.get_info<info::platform::name>();
+    if (platform_name == "Intel(R) OpenCL HD Graphics")
```

Comment:
I believe, there should be a better way to verify the device is either of Intel's GPUs.

Reply:
Right now there is no low-level runtime specification for the feature, so it's the best, what came in my mind. Do you have any ideas?